### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by hamzaw9**
**In this pull request:**

- Delete all the boilerplate from Create React App
- Create a Components directory
- Create a Calculator.js functional component

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,16 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
```
Comment (Alfredbis29) [inline src/index.js:4]: - [ ] I can see that you have made good progress in this project but I can see that you imported the file `index.css` and it does not exist and it brings up an error . kindly fix this for your project to look professional :wink: 
Screenshot from 2023-09-21 16-47-33

File: src/components/style.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  background: linear-gradient(180deg, #2f2f2f, #3f3f3f);
+  box-shadow:
+    inset -8px 0 8px rgba(0, 0, 0, 0.15),
+    inset 0 -8px 8px rgba(0, 0, 0, 0.25),
+    0 0 0 2px rgba(0, 0, 0, 0.75),
+    10px 20px 25px rgba(0, 0, 0, 0.4);
+  border: none;
+  border-radius: 10px;
+  color: whitesmoke;
+}
```
Comment (Alfredbis29) [inline src/components/style.css:61]: - [ ] As required for this project your design should look the same as provided by the team , kindly match your design to the following on the screenshot below, If we compare your design to the one provided by the team we get this :point_down: 

Your Design 
Screenshot from 2023-09-21 16-49-32

The one we should follow 
Screenshot from 2023-09-21 16-51-54

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @hamzaw9 
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark:  Your Work looks Professional 
:heavy_check_mark:  Nice work so far :+1: 
 
 ## Required Changes 

- [ ] Check Under the review 

 ## Optional suggestions 

- [ ] N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @hamzaw9 , 
## STATUS: CHANGES REQUIRED :recycle: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @hamzaw9 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great
-  ✅ Great work using the right GitFlow 
-  ✅ Your README looks good
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**